conftest.py

```diff
@@ -1,3 +1,5 @@
 def pytest_addoption(parser):
     parser.addoption("--runslow", action="store_true",
                      help="run slow tests")
+    parser.addoption("--all-utm-zones", action="store_true",
```

Comment:
instead of running the test with really all utm zones, I wonder if it would make sense to test the edge cases, boundary cases and happy path cases only. This would require more thought on what those cases actually are, but just because we `can` test all cases, doesn't mean we necessarily `need` to (they seem to me quite similar in what they test).